custom/icds_reports/tests/reports/test_awc_reports.py

```diff
@@ -2457,22 +2425,54 @@ def test_awc_report_beneficiary_71230690_c828_4863_b2c1_f61a75aed9d7(self):
                     "recorded_height": 0,
                     "fully_immunized": "No",
                     "person_name": "Name 4407",
-                    "sex": "M",
-                    "mother_name": "ममता बाई"
+                },
+                cls=DjangoJSONEncoder
+            )
+        )
+
+    def test_awc_report_beneficiary_ca040875_2e42_4ce4_acf7_f96695b370f1(self):
+        data = get_awc_report_beneficiary(
+            0, 10, 1, 'dob', 'a18', (2017, 5, 1), (2017, 3, 1), 'icds-cas'
+        )['data'][9]
```

Comment:
Why did the ordering in the list change?

Reply:
@emord I found issue in limit/ordering so in tests the ordering back to normal